marklogic-data-hub/src/main/java/com/marklogic/hub/mapping/MappingFunctions.java

```diff
@@ -0,0 +1,27 @@
+package com.marklogic.hub.mapping;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.extensions.ResourceManager;
+import com.marklogic.client.extensions.ResourceServices;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.util.RequestParameters;
+
+public class MappingFunctions extends ResourceManager {
+    private static final String NAME = "ml:mappingFunctions";
+
+    public MappingFunctions(DatabaseClient client) {
+        super();
+        client.init(NAME, this);
+    }
+
+    public JsonNode getMappingFunctions() {
+        ResourceServices.ServiceResultIterator resultItr = this.getServices().get(new RequestParameters());
```

Comment:
You should be able to simplify this to just the following:

    return getServices().get(new RequestParameters(), new JacksonHandle()).get();